src/css/custom.css

```diff
@@ -225,4 +225,12 @@ table, tr, td, th {
 
 .hard {
   color: rgb(255 45 85);
+}
+
+/*  for emoji's */
+
+@media screen and (min-width: 997px) and (max-width: 1119px) {
```

Comment:
like I said before. don't use between and max-width. Please follow mobile first principle. (i.e. by default no emoji is shown, then show only when min-width is 1200px.

Reply:
okay, I understand, I do to resolve this.

Reply:
Not resolved at all...